module/Company/src/Service/Company.php

```diff
@@ -712,21 +756,23 @@ public function createJob($packageId, $data, $files)
      *
      * @param int|string $languageNeutralId Identifier of the Job to save
      * @param array $jobs The Job to save
-     * @param array $data The (new) data to save
-     * @param array $files The (new) files to save
+     * @param Parameters $data The (new) data to save
+     * @param Parameters $files The (new) files to save
      *
      * @return bool
      */
-    public function saveJobData($languageNeutralId, $jobs, $data, $files)
+    public function saveJobData($languageNeutralId, $jobs, Parameters $data, Parameters $files)
```

Comment:
If you want to change these parameter types from array to Parameters, you should also change the type where the function is called. PhpStan reports such errors. With the actual type outside the PhpDoc, this would probably even throw errors.

Reply:
Stupid PhpDocs :/ All these functions always used `Parameters`, but all DocBlocks mentioned `array`.